cypress.config.ts

```diff
@@ -43,7 +58,10 @@ export default defineConfig({
       on(
         'file:preprocessor',
         createBundler({
-          plugins: [createEsbuildPlugin(config)],
+          plugins: [
+            createEsbuildPlugin(config),
+            createVuePlugin(),
+          ],
```

Comment:
This has a little story behind it:

1. Because our e2e.ts 'main' file pulls in `development.ts` in order to know about the mocks for usage with Cypress's `intercept` it also pulls in a `source` that we decorate for development purposes, despite the fact that we don't need that decorated `source` in Cypress.
2. In order to decorate the `source`, we import the `control-planes` module, as that contains the `source` that we want to decorate.
3. Beginning with this PR we import a `.vue` file from the `control-planes` module, which Cypress then chokes on.

The green above here, essentially turn `.vue` files to export `""` when running in Cypress. We don't actually need the vue files/components in Cypress but just because of how things are currently organized we statically import the `vue` files in Cypress.

---

We should think about splitting of a service container definition file that contains only things about mocks that we can share between browser (MSW), Cypress (intercept) and possibly Express-like servers. But for the moment this approach solves the issue in the short term.